corehq/apps/accounting/forms.py

```diff
@@ -174,6 +179,10 @@ def __init__(self, account, *args, **kwargs):
                     data_bind="checked: is_customer_billing_account",
                 ),
             ))
+            if account.is_customer_billing_account:
```

Comment:
The email list can be shown interactively like:

https://github.com/dimagi/commcare-hq/blob/0abea45ece9c502d5b8fe61dee47ba42ccc75d74/corehq/apps/accounting/forms.py#L599-L602

This way you don't have to refresh the page to see the emails.